src/authentication/githubServer.ts

```diff
@@ -12,8 +12,8 @@ export class GitHubManager {
 			return false;
 		}
 
-		// .wiki repos are not supported
-		if (host.path.endsWith('.wiki')) {
+		// .wiki/.git repos are not supported
+		if (host.path.endsWith('.wiki') || host.path.match(/gist[.]github[.]com/g)) {
```

Comment:
Does this need to be a global regular expression match (`/g`)?

Reply:
I usually go here to visualize regular expressions: https://regexper.com/#%5E%5C%2F%5Ba-z0-9%5D%2B%24

I don't think we want to exclude all github hosts with non-alphanumeric characters in them (I have no idea how often those will occur with enterprise for example). I'm happy with the simple test that this PR uses.

Reply:
I'm ok with ignoring that part and just check the domain name. For that though we need to check `host.authority` instead of `host.path`.

Reply:
Removed the `g` for now

Reply:
@burkeholland, @mcornella is correct, it is `host.authority` then needs to be checked for `gist.github.com`.